ibm_mq/datadog_checks/ibm_mq/channel_metric_collector.py

```diff
@@ -0,0 +1,133 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from six import iteritems
+
+from datadog_checks.base import AgentCheck, ensure_bytes, ensure_unicode
+
+from . import metrics
+
+try:
+    import pymqi
+except ImportError as e:
+    pymqiException = e
+    pymqi = None
+else:
+    # Since pymqi is not be available/installed on win/macOS when running e2e,
+    # we load the following constants only pymqi import succeed
+    STATUS_MQCHS_UNKNOWN = -1
+    CHANNEL_STATUS_NAME_MAPPING = {
+        pymqi.CMQCFC.MQCHS_INACTIVE: "inactive",
+        pymqi.CMQCFC.MQCHS_BINDING: "binding",
+        pymqi.CMQCFC.MQCHS_STARTING: "starting",
+        pymqi.CMQCFC.MQCHS_RUNNING: "running",
+        pymqi.CMQCFC.MQCHS_STOPPING: "stopping",
+        pymqi.CMQCFC.MQCHS_RETRYING: "retrying",
+        pymqi.CMQCFC.MQCHS_STOPPED: "stopped",
+        pymqi.CMQCFC.MQCHS_REQUESTING: "requesting",
+        pymqi.CMQCFC.MQCHS_PAUSED: "paused",
+        pymqi.CMQCFC.MQCHS_INITIALIZING: "initializing",
+        STATUS_MQCHS_UNKNOWN: "unknown",
+    }
+
+
+class ChannelMetricCollector:
```

Comment:
```suggestion
class ChannelMetricCollector(object):
```
Since we are still supporting Python 2

Reply:
good catch!